### DISCUSSION TITLE
Insert into Table Select * from Table

### DISCUSSION THREAD

**Discussion by vwake7** [stackql/stackql#399] [Q&A] [answered]
Thanks @jeffreyaven that really helps. One more related query, if I want to clone a resource. I am not able to do a 'Insert into Table Select * from Table'. I have to do a Select first and then do a subsequent Insert. Is this feature in your backlog list. This will be quite helpful in my case. Thanks.

`query = """
Insert into azure.resources.resource_groups(
	resourceGroupName, 
	subscriptionId, 
	data__location)
select 
	'vmss-flex',
	subscriptionId,
	data__location
from azure.resources.resource_groups;
"""
res = stackql.executeStmt(query)
print(res)`

Error below
`{'message': "cannot find matching operation, possible causes include missing required parameters or an unsupported method for the resource, to find required parameters for supported methods run SHOW METHODS IN azure.resources.resource_groups: no appropriate method = 'select' for resource = 'resource_groups'\n"}`

**Comment by jeffreyaven** [comment, marked as answer]
hey @vwake7 this issue is you aren't supplying the required parameters to the `SELECT `query, try something like:

```sql
Insert into azure.resources.resource_groups( resourceGroupName,  subscriptionId,  data__location) 
select  'vmss-flex', '631d1c6d-2a65-43e7-93c2-688bfe4e1468', location from azure.resources.resource_groups
where subscriptionId = '631d1c6d-2a65-43e7-93c2-688bfe4e1468'
```
`azure.resources.resource_groups` requires `subscriptionId` as a required param, you can see this in `SHOW METHODS IN azure.resources.resource_groups` (check the `select` `SQLVerb` values for `requiredParams`)  

This should work
